INSTALL.md

```diff
@@ -133,6 +133,12 @@ Now you can run Audiogram in a container using that image:
 docker run -p 8888:8888 -t -i audiogram
 ```
 
+If you're using an editor like Sublime or Atom and want file changes to be automatically reflected inside the Docker container, try using Docker volumes. Instead of the above `docker run` command, try:
```

Comment:
Can we rewrite this slightly? How about:

If you're using Docker to run Audiogram, settings changes you make locally won't automatically show up. One way to resolve this is by using a Docker volume, mounting your local Audiogram directory to the container with a command like:

```sh
docker run -v $(pwd):/home/audiogram/audiogram -p 8888:8888 -t -i audiogram
```

See [this issue](https://github.com/nypublicradio/audiogram/issues/39#issuecomment-239681199) for more details.